crates/rattler_conda_types/src/prefix_record.rs

```diff
@@ -85,6 +87,14 @@ pub struct PathsEntry {
     /// The size of the file in bytes
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub size_in_bytes: Option<u64>,
+
+    /// The file mode of the entry. This is used in conjunction with a prefix_placeholder
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub file_mode: Option<FileMode>,
```

Comment:
this adds a number of bytes to the conda-meta output for every path, would it make sense to only serialize this if it is binary?